core-bundle/src/Resources/contao/classes/Frontend.php

```diff
@@ -59,6 +59,8 @@ public function __construct()
 	 */
```

Comment:
The phpDoc is missing a `@deprecated` tag (this is just a note to myself).

Reply:
Added in 847914e2ed2909575f53e21e66a8e9f42ed40004